### ISSUE TITLE
The document can‘t  be download

### ISSUE THREAD

**Issue by zhzg605** [ONLYOFFICE/CommunityServer#286]
Hi，I am a newer to the project，I installed the communityServer,and i put the first document ,but I can't download it,
The mes
![image](https://user-images.githubusercontent.com/2666836/115501288-7f102480-a2a5-11eb-9b21-dacae6564655.png)
sage is :

please help me  hava a look ,Thanks!

**Comment by Carazyda** [comment]
Hello @zhzg605 Please describe what is version do you use, and how you installed it? Also please look at web.files.log for errors.

**Comment by zhzg605** [comment]
Hi,I am install the latest version ：
![image](https://user-images.githubusercontent.com/2666836/115640496-ad474000-a349-11eb-8048-c116062b8def.png)

and the log：
![image](https://user-images.githubusercontent.com/2666836/115640533-c51ec400-a349-11eb-9cf4-3e243fd0b7fa.png)
![Uploading image.png…]()

**Comment by Carazyda** [comment]
How did you install Onlyoffice Document Server, and connect it to Community Server? Please check the JWT signature configuration.

If you want to use a community server integrated with a document server, we recommend use [Workspace Edition](https://www.onlyoffice.com/download.aspx#bundles).

**Comment by zhzg605** [comment]
Hi,I  install Onlyoffice Document Server in another server,and connect it to Community Server use bellow config:
![image](https://user-images.githubusercontent.com/2666836/115799203-8ef54900-a40a-11eb-957d-cbd1763ecd6e.png)

**Comment by zhzg605** [comment]
Hi the  issues is ok, It is because i don't config the url :
![image](https://user-images.githubusercontent.com/2666836/115825836-8e75a600-a43c-11eb-98a3-00d9d91fa427.png)

**Comment by Carazyda** [comment]
Yes, the third field is the community server address for the document server, and therefore it cannot download the document. If the problem is resolved, please close.
